packages/edge-login-ui-react/src/theme/globals.scss

```diff
@@ -1,28 +1,90 @@
 // Global CSS rules that should only be included once
 
-@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
-@import url("https://fonts.googleapis.com/css?family=Source+Sans+Pro");
+@import url('https://fonts.googleapis.com/icon?family=Material+Icons');
+@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro');
+@import url('https://fonts.googleapis.com/css?family=Quicksand:300,400,500,600,700&display=swap');
```

Comment:
How hard would it be to download the fonts and load them as local assets?

Reply:
It's pretty freaking hard with the current toolchain, actually.